configuration/exodus-pipeline.yaml

```diff
@@ -230,6 +232,27 @@ Resources:
                 TemplatePath: BuildArtifact::exodus-lambda-pkg.yaml
               InputArtifacts:
                 - Name: BuildArtifact
+        - !If
+          - NotDev
+          - Name: Test
```

Comment:
I think we should run integration tests before deploying to dev as well.

Reply:
+1